src/main/java/teammates/ui/webapi/output/FeedbackQuestionData.java

```diff
@@ -176,4 +180,18 @@ public Integer getCustomNumberOfEntitiesToGiveFeedbackTo() {
     public List<FeedbackVisibilityType> getShowRecipientNameTo() {
         return showRecipientNameTo;
     }
+
+    /**
+     * Hides weights in question details if present.
+     */
+    public void hideWeightsForStudent() {
```

Comment:
"Weights" is just the specific information that we want to hide. The method itself should be generic, e.g. `hideInformationForStudent` or alike.